packages/lab/src/stepped-tracker/TrackerStep/TrackerStep.tsx

```diff
@@ -19,46 +21,28 @@ import trackerStepCss from "./TrackerStep.css";
 
 const withBaseName = makePrefixer("saltTrackerStep");
 
-type State = "default" | "completed";
-
-type StateWithActive = State | "active";
+type StageOptions = "pending" | "completed";
+type StatusOptions = "warning" | "error" | undefined;
```

Comment:
Do we need `undefined` here? Could we use the ValidationStatus from core?

Reply:
I've updated to remove "default". `ValidationStatus` has "success" and "info" values that have no corresponding icon in this component - seems clearer to stick with `"warning" | "error"` here?

Reply:
You can use `Pick` from Typescript

Reply:
`Pick` doesn't work with the ValidationStatus type (no named properties), but `Extract` does the job